.ci/magician/github/membership.go

```diff
@@ -116,5 +125,21 @@ func GetRandomReviewer() string {
 }
 
 func AvailableReviewers() []string {
-	return utils.Removes(reviewerRotation, onVacationReviewers)
+	var availableReviewers []string
+	for _, id := range reviewerRotation {
```

Comment:
it could be nice (for example for tests) to have onVacation return a list of people currently on vacation instead, and then use utils.Removes to remove them from the rotation

Reply:
Done.

Reply:
This loop seems unnecessary - I think you can loop over onVacationReviewers directly.

Reply:
Done.